Project2.md

```diff
@@ -159,12 +209,34 @@ you'll need to use your Django manage.py to generate a set of SQL commands neede
 your new models. Then you can apply these commands to make them actually take affect. Django breaks this into two
 stages so that you can check the commands into git on your dev machine and then later apply them to many different db's -- in theory you might have many dev db instances, some testing/qa instances and then prod db instances. See the Django getting started or model documentation for more on migrations and how to use them.
 
+Fixtures
+--------
+Final thing before we call it a day, you need to add fixtures so that we are provided with initial data when grading your project.
```

Comment:
Mention that fixtures aren't just about making grading easier, they're about making any sort of testing easier.